### ISSUE TITLE
Event research: How are events being used currently by COA and others?

### ISSUE THREAD

**Issue by toribr** [cityofaustin/techstack#3556]
To begin to uncover a good set of requirements, let's look at how events are utilized elsewhere, and what elements comprise an "event".

[Event overview](https://docs.google.com/document/d/1iQN4Oew8owUuXhcpDglQpuFBeWYeX0vh2aBai0ZN-_U/edit#heading=h.xku6j63ue47n)

**Comment by ablangworthy27** [comment]
I started a [spreadsheet](https://docs.google.com/spreadsheets/u/1/d/1TcKkfMZRVjATZYT6hIBM27FvWLR-wsNLf5pIHAIax8w/edit#gid=0) that looks at some of the different categories we’ve been talking about, broken down into COA events, other cities’ events, and non-gov events. I also have started writing up [summaries](https://docs.google.com/document/d/1QMeSfn9Tq320-In2rdWlAiXxEUk0vcMW3uKZQiRZVjk/edit?folder=15NcQ1eGg64LsnsqGRaF0y-zTDo-sTunm) that are a little more digestible. 

I tried to get a wide variety rather than large amounts of similar events. I’ll keep working on this, but @toribr @chasechenevert let me know if you see something I should focus more on.
